### ISSUE TITLE
Using sudo in subsequent modules after bootstrap doesn't work if you didn't use sudo to bootstrap

### ISSUE THREAD

**Issue by vallard** [defunctzombie/ansible-coreos-bootstrap#13]
following the README.md I didn't use sudo to install the bootstrap and everything was created (pypy, bin) in the /home/core directory.  On subsequent calls, I followed the README and used sudo to install docker-py with pip.  This caused a problem because the /home/core/bin/pip file uses $HOME in the paths and when I used sudo it looked for those files in the /root directory.  

I am using cores 557.2.0 image for openstack (latest I found on their website)

**Comment by hbokh** [comment]
Yep... To get around this I've forked this repo and made sure the Python-stuff is installed as root under /opt/.
See https://github.com/hbokh/ansible-coreos-bootstrap/commit/d0d12acd1d5994b1fee7b5700eecda8bf1fe3fff
The bootstrap-playbook now (also) has to be run with `sudo: true`, just as the example-playbook.

**Comment by vamonte** [comment]
+1. Same issue.

**Comment by defunctzombie** [comment]
This seems to be going back and forth. I remember some PR where root was removed, then new issues started happening. If it helps to document this better in the readme, I would accept a PR for that.
